tools/pipelines/test-service-clients.yml

```diff
@@ -88,3 +88,45 @@ stages:
           odsp__client__login__password: $(odsp-client-login-password)
           odsp__client2__login__username: $(odsp-client2-login-username)
           odsp__client2__login__password: $(odsp-client2-login-password)
+
+  # Capture pipeline stage results
+  - ${{ if eq(parameters.telemetry, true) }}:
+    - stage: upload_run_telemetry
+      displayName: Upload pipeline run telemetry to Kusto
+      condition: succeededOrFailed()
+      dependsOn:
+        - e2e - azure client with frs
+        - e2e - azure client with azure local service
+        - e2e - odsp client with odsp service
```

Comment:
I _think_ we need to add "ids" (instead of just "display names") to each stage, and those are the ones that go here. That's how we set the dependencies for build - client packages (see [here](https://github.com/microsoft/FluidFramework/blob/218b985f5708c781ee31faaf639209c2a0de0fc6/tools/pipelines/templates/include-publish-npm-package.yml#L21-L23)). It's a bit unfortunate that the test pipelines don't run PR checks, we could have seen if this happened to work or not, but I'd suggest let's add the "ids" and use that instead.